pkg/server/api/github.go

```diff
@@ -0,0 +1,122 @@
+package api
```

Comment:
I think the github part should be its own package, either under `api/` or yet somewhere else, since it has nothing to do with current `api` package.

Reply:
Hmm I'm not sure about this, the github portion does portions of the Api struct like the config, logger and db connection. Is it worth redoing these in a new github struct? I don't think it has nothing to do with the current api package. It's an extension to the enroll api

Reply:
I looked again at the `api` package. We could have a GH client that is instantiated with the appropriate resources, like DB client, logger etc., but maybe you're right that for the time being we can just keep everything in the `api` package.